### PR TITLE
docs(renovate): enhance renovate documentation

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: docs/renovate/readme.md [hunk 1]
```diff
@@ -22,10 +21,42 @@ To be able to benefit from the Renovate Presets exposed by the Otter Framework,
 > [!NOTE]
 > The Otter Renovate presets are always prefixed with `github>AmadeusITGroup/otter//tools/renovate/` to target the specific configuration files hosted in the Otter repository.
 
+> [!WARNING]
+> Check the *Self hosted Renovate section* below to see how to fully benefit from the presets.
```
Comment (kpanot) [inline docs/renovate/readme.md:25]:
```suggestion
> Check the [Self hosted Renovate section](#self-hosted-renovate) to see how to fully benefit from the presets.
```

File: docs/renovate/readme.md [hunk 2]
```diff
@@ -22,10 +21,42 @@ To be able to benefit from the Renovate Presets exposed by the Otter Framework,
...
-- **base**: Base configuration recommended to an Otter base project.
-- **otter-project**: Configuration to be used in a project to facilitate and regroup the upgrades of Otter dependencies.
-- **sdk**: Configuration to ensure the upgrade of the dependencies of a generated SDK
-- **sdk-spec-upgrade**: Setup an auto-regeneration of the SDK based on a given Swagger/OAS specification dependency (as first [parameter](https://docs.renovatebot.com/config-presets/#preset-parameters)).
-- **design-factory**: Renovate configuration to ensure the dependency upgrade of the Design Factory dependencies are following the Design Factory constraints.
+You can see here [the documentation of the available presets](https://github.com/AmadeusITGroup/otter/tree/main/tools/renovate).
+
+## Self-Hosted Renovate
+
+The otter presets come with post upgrade commands to help the upgrade process by running migration scripts or regenerating the SDK for example.
```
Comment (cpaulve-1A) [inline docs/renovate/readme.md:33]:
```suggestion
The Otter presets come with post upgrade commands to help the upgrade process by running migration scripts or regenerating the SDK for example.
```

File: docs/renovate/readme.md [hunk 3]
```diff
@@ -22,10 +21,42 @@ To be able to benefit from the Renovate Presets exposed by the Otter Framework,
...
+This feature can only work if these commands are whitelisted.
+Unfortunately, as of today, whitelisting of post upgrade commands is only possible when using a [Self-Hosted Renovate](https://docs.renovatebot.com/examples/self-hosting/).
+
+Instead of using the GitHub App Renovate, you'll need to create your own or to create a workflow to run it.
+The workflow can be as simple as a cron running `npx renovate`.
+
+You will need the following config in your renovate instance to support the otter presets:
```
Comment (cpaulve-1A) [inline docs/renovate/readme.md:40]:
```suggestion
You will need the following config in your renovate instance to support the Otter presets:
```

**Review by kpanot — APPROVED** [review]

**Review by cpaulve-1A — APPROVED** [review]

File: tools/renovate/README.md [hunk 4]
```diff
@@ -7,6 +7,23 @@
 
 Otter framework provides a set of Renovate presets to facilitate the setup and reduce the boilerplate in your `.renovaterc.json`
 
+## Available presets
+
+| Preset                        | Parameters                         | Description                                                                                        |
+|-------------------------------|------------------------------------|----------------------------------------------------------------------------------------------------|
+| **group/design-factory**      |                                    | Group the dependencies related to Design Factory .                                                 |
```
Comment (sdo-1A) [inline tools/renovate/README.md:14]:
```suggestion
| **group/design-factory**      |                                    | Group the dependencies related to Design Factory.                                                 |
```

Comment (sdo-1A) [reply tools/renovate/README.md:14]: Also, to be consistent, either add or remove the "." for each line (not the case currently)

File: tools/renovate/README.md [hunk 5]
```diff
@@ -7,6 +7,23 @@
...
+| **tasks/otter-ng-update**     | package-manager?                   | Trigger the migration scripts when upgrading the Otter dependencies                                |
+| **tasks/sdk-regenerate**      | package-manager?                   | Regenerate the SDK when upgrading the SDK dependencies                                             |
+| **tasks/sdk-spec-regenerate** | package-manager, spec-package-name | Regenerate the SDK when upgrading the SDK spec (when fetching the spec from an NPM repository)     |
+| **tasks/yarn-pnp**            |                                    | **(Yarn only)** Upgrade Yarn SDKs when upgrading the version of Yarn (only relevant with PnP)      |
+| **base**                      |                                    | Base configuration recommended for any project                                                     |
+| **otter-project**             |                                    | **(Yarn only)** Additional configuration recommended for an Otter base project                     |
```
Comment (sdo-1A) [inline tools/renovate/README.md:23]:
```suggestion
| **otter-project**             |                                    | **(Yarn only)** Additional configuration recommended for an Otter-based project                     |
```

File: docs/renovate/readme.md [hunk 6]
```diff
@@ -22,10 +21,42 @@ To be able to benefit from the Renovate Presets exposed by the Otter Framework,
 > [!NOTE]
 > The Otter Renovate presets are always prefixed with `github>AmadeusITGroup/otter//tools/renovate/` to target the specific configuration files hosted in the Otter repository.
 
+> [!WARNING]
+> Check the [Self hosted Renovate section](#self-hosted-renovate) to see how to fully benefit from the presets.
```
Comment (sdo-1A) [inline docs/renovate/readme.md:25]:
```suggestion
> Check out the [Self-Hosted Renovate section](#self-hosted-renovate) to see how to fully benefit from the presets.
```

File: docs/renovate/readme.md [hunk 7]
```diff
@@ -22,10 +21,42 @@ To be able to benefit from the Renovate Presets exposed by the Otter Framework,
...
+
 ## Available presets
 
-- **base**: Base configuration recommended to an Otter base project.
-- **otter-project**: Configuration to be used in a project to facilitate and regroup the upgrades of Otter dependencies.
-- **sdk**: Configuration to ensure the upgrade of the dependencies of a generated SDK
-- **sdk-spec-upgrade**: Setup an auto-regeneration of the SDK based on a given Swagger/OAS specification dependency (as first [parameter](https://docs.renovatebot.com/config-presets/#preset-parameters)).
-- **design-factory**: Renovate configuration to ensure the dependency upgrade of the Design Factory dependencies are following the Design Factory constraints.
+You can see here [the documentation of the available presets](https://github.com/AmadeusITGroup/otter/tree/main/tools/renovate).
```
Comment (sdo-1A) [inline docs/renovate/readme.md:29]:
```suggestion
You can have a look at [the documentation of the available presets](https://github.com/AmadeusITGroup/otter/tree/main/tools/renovate).
```
(avoid using "here" in documentation)

**Pull request merged**